docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,47 @@
+# Troubleshooting
+
+## Compilation issues
+
+### Manifest merger failed
+
+> Manifest merger failed : Attribute application@appComponentFactory value=(android.support.v4.app.CoreComponentFactory) from [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91 is also present at [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86 value=(androidx.core.app.CoreComponentFactory).
```

Comment:
I think you should use code block for this in order to be more "vanilla" 
```
  ```text
  ```
```

Reply:
Problem is that the code block won't wrap the lines, and that means that people will have to scroll to read this. Since this is an error message, a quote block seemed more relevant to me.

Reply:
no problemo